app/views/macros/loadComponentTemplate.njk

```diff
@@ -1,3 +1,3 @@
 {% macro loadComponentTemplate(componentName, params) %}
-  {%- include "../../../src/components/" + componentName + "/template.njk" -%}
+  {%- include "../../../src/govuk/components/" + componentName + "/template.njk" -%}
```

Comment:
Note that you can remove `"../../../src/govuk/components/" + ` and this'll still work.

Reply:
Good spot 👍